.github/workflows/ci.yml

```diff
@@ -90,7 +90,7 @@ jobs:
       env:
         PYTHONIOENCODING: 'utf-8'
     - name: Run unittests
-      run: pytest
+      run: pytest --forked
```

Comment:
FWIW it's best to put this in `pytest.ini` so that it's shared across invocations (and local executions are equivalent to the CI).

Reply:
I also like to have the config annotated like this: https://github.com/cherrypy/cheroot/blob/e1a45fa/pytest.ini, feel free to copy+adapt.